gprofiler/utils/perf_process.py

```diff
@@ -27,7 +27,6 @@ def perf_path() -> str:
     return resource_path("perf")
 
 
-# TODO: automatically disable this profiler if can_i_use_perf_events() returns False?
```

Comment:
You removed it, but is it not relevant anymore?

Reply:
18de021cf661320184a695bd12f43b489735f8dd